packages/manager/modules/pci/src/projects/project/training/jobs/jobs.routing.js

```diff
@@ -7,11 +7,24 @@ export default /* @ngInject */ ($stateProvider) => {
         $translate.instant('pci_projects_project_training_jobs_title'),
       pricesCatalog: /* @ngInject */ (PciProjectTrainingService, projectId) =>
         PciProjectTrainingService.getPricesFromCatalog(projectId),
-      getPrice: /* @ngInject */ (pricesCatalog) => (qty) =>
-        pricesCatalog[`ai-training.ai1-standard.hour.consumption`]
-          .priceInUcents * qty,
-      getTax: /* @ngInject */ (pricesCatalog) => (qty) =>
-        pricesCatalog[`ai-training.ai1-standard.hour.consumption`].tax * qty,
+      getCatalogEntryF: /* @ngInject */ (pricesCatalog) => {
+        return function f(resourceId) {
+          let catalogEntry;
+          if (resourceId === 'gpu') {
+            catalogEntry =
+              pricesCatalog[`ai-training.ai1-1-gpu.minute.consumption`];
+          } else {
+            catalogEntry =
+              pricesCatalog[`ai-training.ai1-1-cpu.minute.consumption`];
+          }
+          return catalogEntry;
+        };
+      },
```

Comment:
```suggestion
      getCatalogEntryF: /* @ngInject */ (pricesCatalog) => (resourceId) => pricesCatalog[`ai-training.ai1-1-${resourceId}.minute.consumption`]
```
Also I'm not sure hardcoding plancode is the best idea for maintenance

Reply:
It is clearly not, and that is something we need to rework